FalconBMS Alternative Launcher Cs/App.xaml.cs

```diff
@@ -1,17 +1,40 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Configuration;
-using System.Data;
-using System.Linq;
-using System.Threading.Tasks;
+using System.Diagnostics;
 using System.Windows;
+using System.Windows.Threading;
 
 namespace FalconBMS_Alternative_Launcher_Cs
 {
     /// <summary>
-    /// App.xaml の相互作用ロジック
+    /// App.xaml
     /// </summary>
-    public partial class App : Application
+    public partial class App
     {
+        public App()
+        {
+            Current.DispatcherUnhandledException += App_DispatcherUnhandledException;
+        }
+
+        private static void App_DispatcherUnhandledException(object sender, DispatcherUnhandledExceptionEventArgs e)
+        {
+            if (e.Exception is NotImplementedException)
+            {
+                MessageBox.Show("This feature has not yet been implemented.", "Feature not Implemented",
+                    MessageBoxButton.OK, MessageBoxImage.Information);
+
+                e.Handled = true;
+            }
+
```

Comment:
Insert additional exception types here as needed for handling known exception scenarios otherwise it goes to the base class Exception.

Reply:
Thank you Hawkeye4040 for the code cleanup and error handling.
I may add an error log output.